### ISSUE TITLE
Feature request: fish support

### ISSUE THREAD

**Issue by kwill** [kislyuk/argcomplete#68]
Would it be possible to also support completions in fish? https://github.com/fish-shell/fish-shell

**Comment by Lucretiel** [comment]
I will see if I can take a stab at this. As of at least 2.0, fish's `complete` method does support specifying a shell function as a custom completer (http://fishshell.com/docs/2.0/commands.html#complete)

**Comment by wernight** [comment]
A simpler alternative may be to generate man page (see https://bugs.python.org/issue14102) and then ask Fish to complete from man page.

**Comment by volkov** [comment]
Following code :
```fish
function __fish_myscript_complete
   set -x _ARGCOMPLETE 1
   set -x _ARGCOMPLETE_IFS \n
   set -x _ARGCOMPLETE_SUPPRESS_SPACE 1
   set -x COMP_LINE (commandline)
   set -x COMP_POINT (commandline -C) 
   myscript 8>&1 9>&2 1>&9 2>&1
end
complete -c myscript -f -a '(__fish_myscript_complete)'
```
in `~/.config/fish/completions/myscript.fish` works fine for me.

**Comment by kislyuk** [comment]
Fish support has been released in v1.10.0, please test.
